internal/api/v20240610preview/generated/models.go

```diff
@@ -559,6 +559,33 @@ type Label struct {
 	Value *string
 }
 
+// LoadBalancerProfile - A representation of the cluster's public load balancer.
+type LoadBalancerProfile struct {
+	// REQUIRED; The desired managed outbound IPs for the cluster's public load balancer
+	ManagedOutboundIPs *ManagedOutboundIPs
+
+	// READ-ONLY; The list of effective outbound IP addresses of the cluster's public load balancer
+	EffectiveOutboundIPs []*string
```

Comment:
Why this name "EffectiveOutboundIPs" the "effective" part seems a bit odd. What about ProvisionedOutboundIPs.

Why is this at this level? Maybe it would make sense in ManagedOutboundIPs. See my other comments about API design.